main.js

```diff
@@ -0,0 +1,97 @@
+var sides = [
+'Miso Glazed Carrots',
+ 'Coleslaw',
+ 'Garden Salad',
+ 'Crisy Potatoes',
+ 'Sweet Potato Tots',
+ 'Coconut Rice',
+ 'Ceasar salad',
+ 'Shrimp Summer Rolls',
+ 'Garlic Butter Mushrooms',
+ 'Hush Puppies',
+ 'Isaac\'s Special'
+]
+var mains = [
+  'Spaghetti and Meatballs',
+  'Pineapple Chicken',
+  'Shakshuka',
+  'Thai Yellow Curry',
+  'Bibimbap',
+  'Chicken Parmesean',
+  'Butternut Squash Soup',
+  'BBQ Chicken Burgers',
+  'Ramen',
+  'Empanadas',
+  'Chicken Fried Rice',
+  'Sheet Pan Fajitas',
+  'Margarita Pizza',
+]
+var desserts = [
+  'Apple Pie',
+  'Lemon Meringue Pie',
+  'Black Forest Cake',
+  'Banana Bread',
+  'Peach Cobbler',
+  'Cheesecake',
+  'Funfetti Cake',
+  'Baklava',
+  'Flan',
+  'Macarons',
+  'Macaroons',
+  'Chocolate Cupcakes',
+  'Pavlova',
+  'Pumpkin Pie',
+  'Key Lime Pie',
+  'Tart Tatin',
+  'Croissants',
+  'Eclairs',
+]
```

Comment:
Let's move our data arrays to a data.js file to clean up and make it easier to use or main.js